### ISSUE TITLE
Un/active button colors in main view 

### ISSUE THREAD

**Issue by clebeaupin** [edrlab/readium-desktop#74]
Currently active color is grey and unactive color is black
Switch this behaviour : grey color for unactive button and black color for active button

**Comment by Scarsniik** [comment]
commit done

**Comment by danielweck** [comment]
Hello @Scarsniik could you please close this issue with a reference (HTTP link) to the Git revision or set of commit hashes that fix the issue? Many thanks!

**Comment by clebeaupin** [comment]
Fixes by https://github.com/edrlab/readium-desktop/commit/e7566ee4cd0cd9bb8e9ba4e5102763668cbc0dbc
